packages/node/package.json

```diff
@@ -25,6 +25,7 @@
     "tslib": "^1.9.3"
   },
   "devDependencies": {
+    "@sentry/tracing": "7.36.0",
```

Comment:
This relationship between these packages was missing and needed to be added to ensure task pipelines know what to do and in what order